src/Catty/DataModel/UserData/UserVariable.swift

```diff
@@ -34,7 +34,7 @@
         }
         set {
             variableLocker.lock()
-            if let value = newValue as? NSString {
+            if let value = truncateIfLengthExceeded(value: newValue as? String) {
```

Comment:
Please do not truncate the value to 160 lines. If necessary, you can truncate the value displayed as a text value, but not the actual variable value. This needs to be larger, e.g., while extracting substrings from a large text using regular expression. Please also add a test for this.